src/main/java/seedu/duke/atomichabit/atomichabit/AtomicHabitManager.java

```diff
@@ -0,0 +1,82 @@
+package seedu.duke.atomichabit.atomichabit;
+import java.util.HashMap;
+import java.util.Scanner;
+
+import seedu.duke.atomichabit.command.*;
+import seedu.duke.command.BadCommandException;
+import seedu.duke.command.CommandParser;
+public class AtomicHabitManager {
+    private final CommandParser parser = new CommandParser();
+    private final String HABIT_DIVIDER = "—---------------------------------------------------------------";
+    private AtomicHabitList habitList;
+    private static String userInput;
+
+    private static Scanner myScanner = new Scanner(System.in);
+
+    /**
+     * Method to be called by MainManager when user wishes to utilise atomichabits
+     */
+    public void run() {
+        start();
+        runCommands();
+    }
+
+    /**
+     * Requests user for input
+     * @return user input in a String
+     */
+    public static String takeInput() {
+        String line = myScanner.nextLine();
+        return line;
+    }
+
+    private void start() {
+        habitList = new AtomicHabitList();
```

Comment:
Avoid initialising variables within instance methods, but put them in the constructor instead(you may define the default no-parameter constructor). That's what the constructor is for, after all, another developer would not be expecting to find the initialisation in an instance method.